lib/fileutils.rb

```diff
@@ -1342,13 +1358,52 @@ def compare_stream(a, b)
   end
   module_function :compare_stream
 
+  # Copies the file entry at path +src+ to the entry at path +dest+;
+  # each of +src+ and +dest+ may be a string or a
+  # {Pathnames}[https://docs.ruby-lang.org/en/master/Pathname.html].
+  #
+  # See {install(1)}[https://man7.org/linux/man-pages/man1/install.1.html].
+  #
+  # If the entry at +dest+ does not exist, copies from +src+ to +dest+:
+  #
+  #   File.read('src0.txt')    # => "aaa\n"
+  #   File.exist?('dest0.txt') # => false
+  #   FileUtils.install('src0.txt', 'dest0.txt')
+  #   File.read('dest0.txt')   # => "aaa\n"
+  #
+  # If +dest+ is a file entry, copies from +src+ to +dest+, overwriting:
   #
-  # If +src+ is not same as +dest+, copies it and changes the permission
-  # mode to +mode+.  If +dest+ is a directory, destination is +dest+/+src+.
-  # This method removes destination before copy.
+  #   File.read('src1.txt')  # => "aaa\n"
+  #   File.read('dest1.txt') # => "bbb\n"
+  #   FileUtils.install('src1.txt', 'dest1.txt')
+  #   File.read('dest1.txt') # => "aaa\n"
+  #
+  # If +dest+ is a directory entry, copies from +src+ to <tt>dest/src</tt>,
+  # overwriting if necessary:
+  #
+  #   File.read('src2.txt')       # => "aaa\n"
+  #   File.read('dest2/src2.txt') # => "bbb\n"
+  #   FileUtils.install('src2.txt', 'dest2')
+  #   File.read('dest2/src2.txt') # => "aaa\n"
+  #
+  # Keyword arguments:
+  #
+  # - <tt>group: <i>group</i></tt> - changes the group.
+  # - <tt>mode: <i>permissions</i></tt> - changes the permissions.
+  # - <tt>noop: true</tt> - does not remove entries; returns +nil+.
+  # - <tt>owner: <i>owner</i></tt> - changes the owner.
+  # - <tt>preserve: true</tt> - preserve timestamps.
+  # - <tt>verbose: true</tt> - prints an equivalent command:
```

Comment:
The format of the keyword arguments is inconsistent here. Some (`noop`, `preserve`, `verbose`) show their default values, which others (`group`, `mode`, `owner`) just repeat the variable name as the value. I think we should just show their default values.

Reply:
The first three you mention show non-default values and their effects.  The other three show values as italicized value-names and their effects;  this is consistent with +mkdir+ and +mkdir_p+ above (whose names I have now changed from _integer_ to _mode_).  Stet?